test/unit/contents_test.rb

```diff
@@ -9,13 +9,33 @@ def test_xml_from_contents
     patient = FHIR.from_contents(xml)
     assert !patient.nil?, 'From contents did not succeed with XML.'
     assert patient.id == 'example', 'Patient did not deserialize correctly.'
+    # check memory
+    before = check_memory
+    patient = nil
+    wait_for_gc
+    after = check_memory
+    unless after.empty?
+      puts "BEFORE GC: #{before}"
+      puts "AFTER GC: #{after}"
+    end
+    assert after.empty?, 'Garbage collection missed FHIR Models.'
```

Comment:
I tried adding them to a function in `test_helper.rb` but it won't work correctly. It has to do with the way Ruby GC deals with variables that were declared further up the execution stack. I broke out the pieces that I could: `check_memory` and `wait_for_gc`... I could put the `unless after empty?` block and assertion in a function, but the variables being set to `nil` has to occur here or the GC won't take them.